magenta-lib/src/main/scala/magenta/deployment_type/DeploymentType.scala

```diff
@@ -17,27 +17,27 @@ trait DeploymentType {
   val paramsList = mutable.Map.empty[String, Param[_]]
   lazy val params = paramsList.values.toSeq
   def perAppActions: PartialFunction[String, DeploymentPackage =>
-    (Lookup, DeployParameters, Stack) => List[Task]]
+    (DeployReporter, Lookup, DeployParameters, Stack) => List[Task]]
```

Comment:
What's the reason for adding this to the start, rather than the end of the list?